rspack.iframe-sdk-embed.config.js

```diff
@@ -49,4 +55,22 @@ module.exports = {
   },
   optimization: { splitChunks: false, runtimeChunk: false },
   devtool: false,
+  plugins: [
+    {
+      name: "copy-embed-js-to-app-path",
+      apply(compiler) {
+        compiler.hooks.afterEmit.tap("copy-embed-js-to-app-path", () => {
+          const tempPath = path.join(OUT_TEMP_PATH, OUT_FILE_NAME);
+          const appPath = path.join(BUILD_PATH, "app/", OUT_FILE_NAME);
+
+          // copy embed.v1.js from the temp directory to the resources directory
+          fs.mkdirSync(path.dirname(appPath), { recursive: true });
+          fs.copyFileSync(tempPath, appPath);
+
+          // cleanup the temp directory to prevent bloat.
+          fs.rmSync(tempPath, { recursive: true });
```

Comment:
This is a workaround for the rspack issue of path collisions and metadata bloat, which causes the JS bundle to be missing on production builds. It has the added benefit of `/app/embed.v1.js` living in the same path of `/app/iframeResizer.js` so it's easier to remember from a user's POV.